variables.tf

```
@@ -169,21 +169,29 @@ variable "restrict_public_buckets" {
 variable "s3_replication_enabled" {
   type        = bool
   default     = false
-  description = "Set this to true and specify `s3_replica_bucket_arn` to enable replication. `versioning_enabled` must also be `true`."
+  description = "Set this to true and specify `replication_rules` to enable replication. `versioning_enabled` must also be `true`."
```

The description references a deprecated variable

```suggestion
  description = "Set this to true and specify `s3_replication_rules` to enable replication. `versioning_enabled` must also be `true`."
```